flags.yml

```diff
@@ -40,3 +40,11 @@
   default: false
   contact: Lyon Hill
   expose: true
+
+- name: RedirectTo Path After Login
```

Comment:
is there a corresponding idpe PR that matches this one?

Reply:
no PR up yet. I'm glad you commented on this. When we talked to @gavincabbage about it on Friday it seemed like the UI yml file would generate the flag for the API too, is that right? Right now we want to write up some tests for the API to make sure everything is smooth before submitting a PR for cloud

Reply:
it will show up on `api/v2/flags` if that's what you're referring to as the API (we need to figure out better words for all this), but only for people who are hosting this locally, which seems like the opposite of the intent of this PR? If you want the flag to show up on a production deployment, it needs to be made on IDPE. You should probably even remove the flag from this repo, as local overrides would work for development and the flag isn't meant for OSS instances that handle their own auth.

Reply:
Gotcha. So I'll remove the local flag and set it in IDPE

Reply:
Some clarifying details: The flag only needs to be in one place, either `influxdb/flags.yml` or `idpe/flags.yml`. The former for flags used by OSS or both OSS and IDPE. The latter for flags used by only IDPE. Since the UI lives in OSS afaiu, most or all UI related flags will probably end up here.

Reply:
I made the above comment without refreshing this page, and missed @drdelambre's latest comment.

If the code is resilient to the flag only being present in IDPE's `api/v2/flags` response and not in OSS's, then moving the flag to IDPE also works.